src/env.h

```diff
@@ -379,6 +379,8 @@ constexpr size_t kFsStatsBufferLength =
   V(async_wrap_ctor_template, v8::FunctionTemplate)                            \
   V(async_wrap_object_ctor_template, v8::FunctionTemplate)                     \
   V(compiled_fn_entry_template, v8::ObjectTemplate)                            \
+  V(dir_constructor_template, v8::ObjectTemplate)                              \
```

Comment:
Can you name this `dir_instance_template`? We’re not great on getting this right inside this list, but this is the instance template and not the constructor’s function template